docs/explanations/how_to_interpret_output_of_fix_plan_api.md

```diff
@@ -157,6 +165,44 @@ The JSON representation for this step is:
 The `data` object will state why no operation is needed. This can either be because the CVE doesn't
 affect the system, or because the CVE is already fixed in the machine.
 
+### NoOpAlreadyFixed
+
+This is a subtype of a NoOp step that indicates that the CVE is already
+fixed in the machine. The JSON representation for this step is:
+
+```json
+{
+  "data": {
+    "status": "cve-already-fixed"
+    "pocket": "standard-updates",
+    "source_packages": ["source_package"]
+  },
+  "operation": "no-op",
+  "order": 1
+}
+```
+
+The `data` object will state which packages are already fixed and which
+pocket they come from.
+
+### NoOpLivepatchFixData
+
+This is a subtype of a NoOp step that indicates that the CVE is already
+fixed by a Livepatch patch. The JSON representation for this step is:
```

Comment:
I would suggest "fixed by a patch from Livepatch" - "Livepatch patch" sounds really odd

Reply:
I agree that is better